### ISSUE TITLE
oktaAuth.signOut promise never resolves

### ISSUE THREAD

**Issue by kellengreen** [okta/okta-react#174]
The `oktaAuth.signOut` promise never resolves after visiting a broken `<LoginCallback>` view.

## I'm submitting a:

- [x] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior
`oktaAuth.signOut()` never resolves.

## Expected behavior
`oktaAuth.signOut()` should either resolve or reject.

## Minimal reproduction of the problem with instructions
Demo and steps to reproduce are located [here](https://github.com/kellengreen/okta#reproduce-logout-promise-hangs-bug).

## Environment

```
    "@okta/okta-auth-js": "5.8.0",
    "@okta/okta-react": "6.3.0",
    "react": "^17.0.2",
```

**Comment by denysoblohin-okta** [comment]
@kellengreen 
I've tried to reproduce this issue with your [demo app](https://github.com/kellengreen/okta)
I found that this bug was present in okta-auth-js 5.5.0
But it was fixed in version 5.6.0 with [this PR](https://github.com/okta/okta-auth-js/pull/922)
I noticed that you used version 5.5.0 first in demo and then upgraded to 5.8.0 in [this commit](https://github.com/kellengreen/okta/commit/f0e3c421d6a9fbc904b7d39e275c7f3e4237e1ad#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L28)
Can you please confirm that promise `oktaAuth.signOut()` can be resolved after upgrade?

**Comment by kellengreen** [comment]
Sure thanks for looking. I'll take a peak on Monday.

**Comment by kellengreen** [comment]
Yep it looks like 174 was fixed in `5.6.0+` 👍. 

I retested [173](https://github.com/okta/okta-react/issues/173) with latest versions and I'm still able to reproduce.
